pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -290,15 +290,15 @@ const PageSystemInformationChangeHostname = () => {
                 </FormGroup>
                 <FormGroup fieldId="sich-hostname" label={_("Real host name")}>
                     <TextInput id="sich-hostname" value={hostname} onChange={(_event, value) => onHostnameChanged(value)} validated={error.length ? "error" : "default"} />
-                    <FormHelperText isHidden={error.length == 0} component="div">
+                    {error.length > 0 && <FormHelperText>
                         <HelperText>
                             {error.map((err, i) =>
                                 <HelperTextItem key={i} variant="error">
                                     {err}
                                 </HelperTextItem>
                             )}
                         </HelperText>
-                    </FormHelperText>
+                    </FormHelperText>}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18834-20230523-091737-c1673f0a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)